win_build_vs.bat

```diff
@@ -65,34 +63,33 @@ for %%x in (%*) do (
     if [%%~x] EQU [/T]                set skiptests=ON
     if [%%~x] EQU [/R]                set skiptestrun=ON
     if [%%~x] EQU [/P]                set cpack=ON
-    if [%%~x] EQU [/K]                set build_kafka=ON
-    if [%%~x] EQU [/J]                set build_JNI=ON
-    if [%%~x] EQU [/S]                set build_SQL=ON
-    if [%%~x] EQU [/C]                set build_coap=ON
-    if [%%~x] EQU [/A]                set build_AWS=ON
-    if [%%~x] EQU [/SFTP]             set build_SFTP=ON
-    if [%%~x] EQU [/PDH]              set build_PDH=ON
-    if [%%~x] EQU [/SPLUNK]           set build_SPLUNK=ON
-    if [%%~x] EQU [/GCP]              set build_GCP=ON
-    if [%%~x] EQU [/ELASTIC]          set build_ELASTIC=ON
+    if [%%~x] EQU [/NO_KAFKA]         set enable_kafka=OFF
+    if [%%~x] EQU [/J]                set enable_jni=ON
+    if [%%~x] EQU [/NO_SQL]           set enable_sql=OFF
+    if [%%~x] EQU [/C]                set enable_coap=ON
+    if [%%~x] EQU [/NO_AWS]           set enable_aws=OFF
+    if [%%~x] EQU [/PDH]              set enable_pdh=ON
+    if [%%~x] EQU [/NO_SPLUNK]        set enable_splunk=OFF
+    if [%%~x] EQU [/NO_GCP]           set enable_gcp=OFF
+    if [%%~x] EQU [/NO_ELASTIC]       set enable_elastic=OFF
     if [%%~x] EQU [/M]                set installer_merge_modules=ON
-    if [%%~x] EQU [/Z]                set build_azure=ON
-    if [%%~x] EQU [/N]                set build_nanofi=ON
-    if [%%~x] EQU [/O]                set build_opencv=ON
-    if [%%~x] EQU [/PR]               set build_prometheus=ON
+    if [%%~x] EQU [/NO_AZURE]         set enable_azure=OFF
+    if [%%~x] EQU [/N]                set enable_nanofi=ON
+    if [%%~x] EQU [/O]                set enable_opencv=ON
+    if [%%~x] EQU [/NO_PROMETHEUS]    set enable_prometheus=OFF
     if [%%~x] EQU [/BUSTACHE]         set enable_bustache=ON
     if [%%~x] EQU [/COAP]             set enable_coap=ON
-    if [%%~x] EQU [/ENCRYPT_CONFIG]   set enable_encrypt_config=ON
-    if [%%~x] EQU [/LUA_SCRIPTING]    set enable_lua_scripting=ON
-    if [%%~x] EQU [/MQTT]             set enable_mqtt=ON
-    if [%%~x] EQU [/OPC]              set enable_opc=ON
+    if [%%~x] EQU [/NO_ENCRYPT_CONFIG] set enable_encrypt_config=OFF
+    if [%%~x] EQU [/NO_LUA_SCRIPTING] set enable_lua_scripting=OFF
+    if [%%~x] EQU [/NO_MQTT]          set enable_mqtt=OFF
+    if [%%~x] EQU [/NO_OPC]           set enable_opc=OFF
     if [%%~x] EQU [/OPENWSMAN]        set enable_openwsman=ON
-    if [%%~x] EQU [/OPS]              set enable_ops=ON
+    if [%%~x] EQU [/NO_OPS]           set enable_ops=OFF
     if [%%~x] EQU [/PCAP]             set enable_pcap=ON
-    if [%%~x] EQU [/PYTHON_SCRIPTING] set enable_python_scripting=ON
+    if [%%~x] EQU [/NO_PYTHON_SCRIPTING] set enable_python_scripting=OFF
```

Comment:
what crossed my mind is if we could mirror the arguments like `/std:c++20`, as I understand they are still treated as a single token, but they provide something like a key-value argument, having a "NO_" prefix looks like a strong coupling between how we parametrize `win_build_vs.bat` and the default extensions, while `/MQTT:NO` feels different

Reply:
I want to keep the changes as minimal as possible. Would `/MQTT` (meaning on) and `/MQTT:OFF` (meaning off) be enough for all features?

Reply:
I didn't know it will be replaced soon, in that case it is fine as it is

Reply:
[MINIFICPP-1797 Python bootstrap rewrite](https://issues.apache.org/jira/browse/MINIFICPP-1797)